File: src/main/java/com/rukka/guess/Game.java
package com.rukka.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int secretNum = random.nextInt(10) + 1;
//        secretNum = (int)(Math.random() * 10) + 1;
        System.out.println("secret num: " + secretNum);
        Scanner scanner = new Scanner(System.in);
        int num = 0;
        while (num != secretNum) {
            System.out.print("Please enter a number: ");
            num = scanner.nextInt();
            if (num == secretNum) {
                System.out.println("Great! You got it.");
            } else if (num < secretNum) {
                System.out.println("higher");
            } else {
                System.out.println("lower");
            }
//            System.out.println(num);
        }
    }
}

File: src/main/kotlin/com/rukka/Game.kt
package com.rukka

import java.util.*
import kotlin.random.Random


fun main() {
    val secretNum = Random.nextInt(10) +1
    println("secret num -> $secretNum")
//    val scanner = Scanner(System.`in`)
    var num = 0
    while (num != secretNum) {
        print("Please enter a number: ")
//        num = scanner.nextInt()
        num = readLine()!!.toInt()
        if (num > secretNum) {
            println("lower")
        } else if (num < secretNum) {
            println("higher")
        } else {
            println("Great! You got it.")
        }
    }
}